packages/hardhat/contracts/test/Reentrant.sol

```diff
@@ -0,0 +1,52 @@
+//SPDX-License-Identifier: Unlicense
```

Comment:
This attacker is not really functional, but it can show that any attempts to re-enter will revert.